apis/apiextensions/v1/composition_types.go

```diff
@@ -47,6 +47,11 @@ const (
 	errFmtTransformTypeFailed          = "%s transform could not resolve"
 	errFmtMapTypeNotSupported          = "type %s is not supported for map transform"
 	errFmtMapNotFound                  = "key %s is not found in map"
+
+	errConstantValue                 = "type %s requires constantValue to be set"
+	errConstantValueTypeNotDefined   = "constantValue type not defined"
+	errConstantValueTypeNotSupported = "constantValue type %s is not supported"
+	errRequiredValue                 = "constant value for for type %s is missing"
```

Comment:
Nit: We have a convention of using an `errFmt` prefix for anything that is a format string, rather than a plain string.